docs/source/release/v5.0.0/index.rst

```diff
@@ -4,21 +4,25 @@
 Mantid 5.0.0 Release Notes
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../images/Py3Python.png
    :class: screenshot
-   :width: 385px
+   :width: 500px
    :align: right
 
 .. contents:: Table of Contents
    :local:
 
-.. warning:: This release is still under construction. The changes can be found in the nightly builds on the `download page`_.
-
 We are proud to announce version 5.0.0 of Mantid.
 
-**TODO: Add paragraph summarizing big changes**
+**MANTID NOW RUNS IN Python 3**
+
+Make sure you update any of your Mantid scripts that are currently written in Python 2.
+As a reminder this migration is because Python 2 support ended on 1st Jan 2020.
+
+For this release we have fully  migrated all scripts in the Mantid Script Repository to be Python 3 compatible. Other highlights include the addition of *Waterfall Plots* to Workbench,
```

Comment:
Could we add a couple more workbench highlights to show off a bit more stuff? 

One thing mentioned by a lot of scientists is that the default plot style is lines and not points. It would be good to say that Workbench now has options to control the defaults of a few aspects of the plots along with a much more comprehensive individual plot config menu. It's probably worth mentioning there have been several performance improvements made too. Maybe make this a bullet list?